_pages/user-experience/knowledge-articles.md

```diff
@@ -15,9 +15,54 @@ sidenav:
     href: "user-experience/failure-proof/"
   - text: Knowledge articles
     href: "user-experience/knowledge-articles/"
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## Knowledge Articles
+## Prepare customer support materials
+
+As you prepare to launch your integration with Login.gov, it is important to consider how your agency will support new users. Here are some suggestions to consider:
+
+<ul>
+  <li>
+    Announce the upcoming change on your site and include a time frame rather than exact date.
+  </li>
+  <li>
+    Link to the <a
+    class="usa-link usa-link--external"
+    rel="noreferrer"
+    target="_blank"
+    href="https://www.login.gov/create-an-account/"
+    >Create an account</a>
+    page at Login.gov so users can read more about creating an account.
+  </li>
+  <li>
+    Include contact information for your help desk or Login.gov customer support on a dedicated help page.
+  </li>
+  <li>
+    Be sure to communicate the go-live date with the Login.gov customer support center so agents are prepared for a spike in new account creation support cases.
+  </li>
+</ul>
+
+Some partners have created knowledge base articles that they post on their website, send to users directly by the agency’s customer support team, or forward to the Login.gov customer support team. These can be especially helpful if your integration includes any additional authentication steps on your agency's side. Here are some topics for knowledge articles:
+
+- A brief step-by-step guide to the Login.gov authentication process and the identity verification (IdV) if your agency supports IdV.
+- Short description of each requirement in the authentication process, i.e. multi-factor authentication or using a unique email.
+- Which support topics should the user contact your agency for help with vs. Login.gov.
+- If existing users will need to create a new account or if they will be able to link their agency account to the Login.gov account.
+
+The Login.gov customer support team can assist users with account creation questions, but they cannot support any agency-specific issues. It is important that users are able to find the correct and timely help. Consider including a few support examples along with the customer support contact information. For example:
```

Comment:
Suggested change - 
"The Login.gov customer support team can assist users with account creation questions, but they cannot support any agency-specific issues. It is important that users are able to find correct and timely help. Consider including a few support examples along with the link to the Login.gov help center [https://www.login.gov/help/]. For example:"